src/Controller/ArtCardController.php

```diff
@@ -123,6 +123,7 @@ public function show( ArtCard $artCard ): Response
 
         return $this->render('artCard/show.html.twig', [
             'artCard' => $artCard,
+            'id' => $artCard->getId()
```

Comment:
No you shouldn't have to pass the id here as you already pass the whole artCard to Twig